apps/src/templates/CollapsibleSection.tsx

```diff
@@ -0,0 +1,55 @@
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import Typography from '@cdo/apps/componentLibrary/typography/Typography';
+import React, {useState} from 'react';
+import moduleStyles from './collapsible-section.module.scss';
+import {
+  SemanticTag,
+  VisualAppearance,
+} from '@cdo/apps/componentLibrary/typography/types';
+
+const CollapsibleSection: React.FunctionComponent<{
```

Comment:
nit: I know I had put the prop type definition inline (due to hastiness 🙂), but would you mind moving this out into a `CollapsibleSectionProps` definition?